client/src/assets/scss/styles.scss

```diff
@@ -0,0 +1,379 @@
+@import './_variables.scss';
+@import 'node_modules/bootstrap/scss/bootstrap.scss';
+@import './material.css';
+@import 'node_modules/font-awesome/scss/font-awesome.scss';
+@import './common.scss';
+
+body {
+    font-size: 14px;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+    background-color: rgba(0,0,0,0.02);
+  }
+
+  *{
```

Comment:
## Important
`*` this is a wildcard and it's known to drop performance. This is best avoided.